spec/spec_helper.rb

```diff
@@ -91,4 +91,12 @@
   config.profile_examples = 10
 =end
   # rubocop:enable Style/BlockComments
+
+  def sign_in(user)
+    session[:user_id] = user.try(:id)
```

Comment:
I'm not a big fan of `try`. Is there ever a case where `sign_in` should be called with a nil user? Should that be an exception?